packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                },
+            },
+            {
+                id: 'deliveryScheduled',
+                label: 'Delivery scheduled',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="xxs">
+                                <Text fz={12} fw={500} color="gray.6">
+                                    {formatTime(
+                                        currentLogs(item, logs)[0]
+                                            .scheduledTime,
+                                    )}
+                                </Text>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        handleScheduledDeliveryLogs(
+                                            item,
+                                            logs,
+                                        )[0].scheduledTime,
+                                    )}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        sendNotificationLogs(item, logs)[0]
+                                            .scheduledTime,
+                                    )}
+                                </Text>
+                            </Collapse>
+                        </Box>
+                    ) : (
+                        <Text fz={12} color="gray.6">
+                            -
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'deliveryStarted',
+                label: 'Delivery start',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="xxs">
+                                <Text fz={12} fw={500} color="gray.6">
+                                    {formatTime(
+                                        currentLogs(item, logs)[0].createdAt,
+                                    )}
+                                </Text>
+                            </Group>
+                            <Collapse in={opened}>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
+                                    {formatTime(
+                                        handleScheduledDeliveryLogs(
+                                            item,
+                                            logs,
+                                        )[0].createdAt,
+                                    )}
+                                </Text>
+                                <Text fz={12} fw={500} pt="md" color="gray.6">
```

Comment:
both of those are wrapped in a same text component. you can have one component (for example Collapse) which will apply font and color properties and two <Text elements without props and they will inherit those font styles from the parent